app/javascript/components/common/GraphicalIcon.tsx

```diff
@@ -5,8 +5,8 @@ export function GraphicalIcon({
   icon,
   className = '',
   category,
-  width = null,
-  height = null,
+  width = 0,
+  height = 0,
```

Comment:
The default width and height are zero?

Reply:
yes, this is just for TS. it was `null`, and it was constantly red for me, so changed it to a number.